bundle/manifests/quay-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,285 @@
+        - path: conditions
+          displayName: Conditions
+          description: Observed conditions of Quay components.
+          x-descriptors:
+            - 'urn:alm:descriptor:io.kubernetes.conditions'
+        - path: configEditorCredentialsSecret
+          displayName: Config Editor Credentials Secret
+          description: Name of the secret containing credentials for the Quay config editor.
+          x-descriptors:
+            - 'urn:alm:descriptor:io.kubernetes:Secret'
+        - path: registryEndpoint
+          displayName: Registry Endpoint
+          description: Externally accessible URL for container pull/push and web frontend.
+          x-descriptors:
+            - 'urn:alm:descriptor:org.w3:link'
+        - path: configEditorEndpoint
+          displayName: Config Editor Endpoint
+          description: Externally accessible URL for the config editor UI.
+          x-descriptors:
+            - 'urn:alm:descriptor:org.w3:link'
+    - kind: QuayEcosystem
+      version: v1alpha1
+      name: quayecosystems.redhatcop.redhat.io
+      displayName: Quay Ecosystem
+      description: "[DEPRECATED] Old representation of a full Quay installation."
+  description: Opinionated deployment of Quay on Kubernetes.
+  displayName: Quay
+  install:
+    spec:
+      deployments:
+      - name: quay-operator-tng
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: quay-operator-alm-owned
+          template:
+            metadata:
+              labels:
+                name: quay-operator-alm-owned
+              name: quay-operator-alm-owned
+            spec:
+              containers:
+              - name: quay-operator
+                image: quay.io/projectquay/quay-operator:v3.6.1
+                command:
+                - /workspace/manager
+                - '--namespace=$(WATCH_NAMESPACE)'
+                env:
+                - name: MY_POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: MY_POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: QUAY_VERSION
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['quay-version']
+                - name: QUAY_DEFAULT_BRANDING
+                  value: upstream
+                - name: RELATED_IMAGE_COMPONENT_QUAY
+                  value: quay.io/projectquay/quay:latest
+                - name: RELATED_IMAGE_COMPONENT_CLAIR
+                  value: quay.io/projectquay/clair:nightly
+                - name: RELATED_IMAGE_COMPONENT_BUILDER
```

Comment:
Builder and builder qemu components we previously _not_ in upstream versions of our CRD. I'm not sure why, so for simplicity sake I've added them.